packages/driver/test/cypress/integration/commands/actions/type_spec.coffee

```diff
@@ -77,6 +77,15 @@ describe "src/cy/commands/actions/type", ->
         .then ->
           expect(blurred).to.be.true
 
+    it "limits text entered to the maxlength attribute of a text input", ->
```

Comment:
This is great. Could you add another test covering an invalid maxlength like `maxlength="five"`?